### PR TITLE
Update install requirements

### PR DESCRIPTION
**Pull request by CodyCBakerPhD**
While checking out #47, noticed some missing dependencies for instantiating the converter

### CODE REVIEW

File: tools/anaconda-env/environment-Linux.yml [hunk 1]
```diff
@@ -24,6 +24,6 @@ dependencies:
       - gevent == 21.12.0
       - werkzeug == 2.0.2
       - opencv-python == 4.5.3.56
-      - neuroconv @ git+https://github.com/catalystneuro/neuroconv.git@main
+      - neuroconv @ git+https://github.com/catalystneuro/neuroconv.git@main[ecephys]
```
Comment (CodyCBakerPhD) [inline tools/anaconda-env/environment-Linux.yml:27]:
```suggestion
      - neuroconv @ git+https://github.com/catalystneuro/neuroconv.git@main#neuroconv[ecephys]
```

File: tools/anaconda-env/environment-MAC.yml [hunk 2]
```diff
@@ -24,6 +24,6 @@ dependencies:
       - protobuf == 3.20.0
       - werkzeug == 2.0.2
       - opencv-python == 4.5.3.56
-      - neuroconv @ git+https://github.com/catalystneuro/neuroconv.git@main
+      - neuroconv @ git+https://github.com/catalystneuro/neuroconv.git@main[ecephys]
```
Comment (CodyCBakerPhD) [inline tools/anaconda-env/environment-MAC.yml:27]:
```suggestion
      - neuroconv @ git+https://github.com/catalystneuro/neuroconv.git@main#neuroconv[ecephys]
```

File: tools/anaconda-env/environment-Windows.yml [hunk 3]
```diff
@@ -27,6 +27,6 @@ dependencies:
       - PyInstaller == 4.7
       - opencv-python == 4.5.3.56
       - certifi == 2021.10.8
-      - neuroconv @ git+https://github.com/catalystneuro/neuroconv.git@main
+      - neuroconv @ git+https://github.com/catalystneuro/neuroconv.git@main[ecephys]
```
Comment (CodyCBakerPhD) [inline tools/anaconda-env/environment-Windows.yml:30]:
```suggestion
      - neuroconv @ git+https://github.com/catalystneuro/neuroconv.git@main#neuroconv[ecephys]
```

**Review by garrettmflynn — APPROVED** [review]
This is great. Thank you for catching this! 

As we scale up the number of interfaces, will this be the same approach we take? Or is there a reason/way to resolve optional dependencies as we need them?

**Pull request merged**